core-tests/shared/src/test/scala/zio/CauseSpec.scala

```diff
@@ -168,6 +168,120 @@ object CauseSpec extends ZIOBaseSpec {
         val stripped = cause.stripSomeDefects { case _: NumberFormatException => }
         assert(stripped)(isNone)
       }
+    ),
+    suite("filter")(
```

Comment:
Can you remove the Cause changes that are not related to this PR?

Reply:
funny, thought I branched out of the series/2.x branch,
even funnier, these are already merged